docs/content/docs/wip/ingress.md

```diff
@@ -0,0 +1,427 @@
+
+## Sample demo
+
+### HTTP traffic with ingress
+
+The following demo sends a request from an external IP to a httpbin service inside the cluster. 
+
+1. Install OSM.
+    ```bash
+    osm install
+    ```
+
+1. Install the [nginx ingress controller](https://kubernetes.github.io/ingress-nginx/deploy/#installation-guide)
+
+1. Deploy the `httpbin` service into the `httpbin` namespace after enrolling its namespace to the mesh. The `httpbin` service runs on port `14001`. 
+    ```bash
+    # Create the httpbin namespace
+    kubectl create namespace httpbin
+
+    # Add the namespace to the mesh
+    osm namespace add httpbin
+
+    # Deploy httpbin service in the httpbin namespace
+    kubectl apply -f docs/example/manifests/samples/httpbin/httpbin.yaml -n httpbin
+    ```
+
+    Confirm the `httpbin` service and pods are up and running.
+
+    ```console
+    $ kubectl get pods -n httpbin
+    NAME                       READY   STATUS    RESTARTS   AGE
+    httpbin-74677b7df7-zzlm2   2/2     Running   0          11h
+    ```
+
+    ```console
+    $ kubectl get svc -n httpbin
+    NAME      TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)     AGE
+    httpbin   ClusterIP   10.0.22.196   <none>        14001/TCP   11h
+    ```
+
+1. Apply an ingress configuration yaml to expose the HTTP path `/status/200` on the `httpbin` service with `kubectl apply -f`
+    ```yaml
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+    name: httpbin-ingress
+    namespace: httpbin
+    annotations:
+        kubernetes.io/ingress.class: nginx
+    spec:
+    rules:
+    - http:
+        paths:
+        - path: "/status/200"
+            backend:
+            serviceName: httpbin
+            servicePort: 14001
+    ```
+
+    Confirm that the httpbin-ingress has been successfully deployed. 
+    
+    ```console
+    $ kubectl get ingress -n httpbin
+    NAME              CLASS    HOSTS   ADDRESS         PORTS   AGE
+    httpbin-ingress   <none>   *       20.72.132.186   80      11h
+    ```
+
+1. Confirm that a request to the httpbin service from the external IP address of the Ingress resource succeeds (in this case, the external address would be `20.72.132.186`)
+
+    ```bash
+    curl http://<external-ip>/status/200
+    ```
+
+1. Update the existing ingress with a host specified by applying the following yaml, and confirm that the request succeeds:  
+
+    ```yaml
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+    name: httpbin-ingress
+    namespace: httpbin
+    annotations:
+        kubernetes.io/ingress.class: nginx
+    spec:
+    rules:
+    - host: httpbin.com 
+        http:
+        paths:
+        - path: /status/200
+            backend:
+            serviceName: httpbin
+            servicePort: 14001
+    ```
+
+    ```bash
+    curl http://<external-ip>/status/200 -H "Host: httpbin.com"
+    ```
+
+
+## Other ingress configuration examples
```

Comment:
This title is ambiguous, can it be made more specific?
Ex. `Sample ingress configurations`